script.js

```diff
@@ -39,9 +136,9 @@ function checkFormValue() {
   } else {
     let error = document.createElement('p');
     error.className = 'error';
-    slides[0].prepend(error);
+    document.querySelector('.content').prepend(error);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
    document.querySelector(".content").prepend(error);
```